src/addons/chain.rs

```diff
@@ -1,5 +1,84 @@
+use crate::local_command::LocalCommand;
+use crate::{Process, ProcessCompleted, ProcessError};
 use bevy::prelude::*;
 
-// Coming soon.
-#[derive(Debug, Component)]
-pub enum Chain {}
+#[derive(Component)]
+pub struct Chain {
+    pub(crate) commands: Vec<LocalCommand>,
+}
+
+impl Chain {
+    pub fn new(commands: Vec<LocalCommand>) -> Self {
```

Comment:
Can you make commands here something like `impl IntoIter<Item = LocalCommand>`?

I think this can yield multiple advantages:

- The user can provide a slice of commands if all the commands are known upfront, simplifying the usage
- We can change the internal storage type later, we may want this to make the command remov more efficient later on

Reply:
Good idea. Done.